app/models/settings.rb

```diff
@@ -27,6 +27,10 @@ def self.columns
     super.reject {|column| /\Aheroku_(id|name)|log_requests_switch\Z/ =~ column.name }
   end
 
+  def self.non_null_columns_names
+    columns.select { |column| column.null == false }.map(&:name)
```

Comment:
```suggestion
    columns.select { |column| !column.null }.map(&:name)
```
Or do we want to handle `nil` differently?

Reply:
Honestly, I don't know if `@null` attribute can be `nil` or not. By default, it's set to `true` if missing https://github.com/rails/rails/blob/v6.1.7.7/activerecord/lib/active_record/connection_adapters/column.rb#L20, but I think nothing prevents it from being set to `nil` explicitly.

I think normally both would work the same way.

Reply:
`!column.null` makes more sense then, it's something that I would expect to see as an expression while `==`  provokes the question, why was such expression being used.